src/App.tsx

```diff
@@ -34,62 +35,127 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.ALPABET:
+        return good1.localeCompare(good2);
+
+      case SortType.LENGTH:
+        return good1.length - good2.length;
+
+      default:
```

Comment:
```suggestion
      case SortType.NONE
```
add all known cases

Reply:
I'll add. But I don't understand why? I have cases in which I do different actions ... Why is it necessary to list everything if I will do in these cases what I do by default?

Reply:
cases should cover 100% of actions that you can pass in `sortType`. If you will not do this and no case works out, your switch returns `undefined` and that can cause bugs.